src/GraphQL/Introspection/__DirectiveValue.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL.Types;
+using GraphQL.Utilities;
+
+namespace GraphQL.Introspection
+{
+    public class __DirectiveValue : ObjectGraphType<KeyValuePair<string, SchemaDirectiveVisitor>>
+    {
+        public __DirectiveValue()
+        {
+            Name = nameof(__DirectiveValue);
+            Description =
+                "Schema Directive applied to a element";
```

Comment:
"Schema Directive applied to a element"
->
"Directive applied to a schema element"